Bio/PDB/PDBList.py

```diff
@@ -162,33 +173,68 @@ def get_all_obsolete(self):
             # to include an assert to check for mis-reading the data.
             obsolete = []
             for line in handle:
-                if not line.startswith("OBSLTE "):
+                if not line.startswith(b"OBSLTE "):
                     continue
                 pdb = line.split()[2]
                 assert len(pdb) == 4
                 obsolete.append(pdb)
         return obsolete
 
-    def retrieve_pdb_file(self, pdb_code, obsolete=False, pdir=None):
+    def retrieve_pdb_file(self, pdb_code, file_format='mmCif', overwrite=False, obsolete=False, pdir=None):
```

Comment:
By changing the argument order your have broken backward compatibility, calls written as ``.retrieve_pdb_file(a, b, c)`` should still be interpreted as ``obsolete=b`` and ``pdir=c``.

Add the new arguments to the end, i.e.

``` python
def retrieve_pdb_file(self, pdb_code, obsolete=False, pdir=None, file_format='mmCif', overwrite=False):
```